File: python-validator-olympics/src/question.py
from typing import List
from src.test import Test

class Question:

    def __init__(self, name: str, tests: List[Test]) -> None:
        self.__name = name
        self.__tests = tests

    def get_name(self) -> str:
        return self.__name

    def get_tests(self) -> List[Test]:
        return self.__tests

File: validator/javascript/src/question.js
class Question {

    constructor(name, tests) {
        this.name = name;
        this.tests = tests;
    }
    
    getName() {
        return this.name;
    }

    getTests() {
        return this.tests;
    }
}

    
module.exports = {
   Question
}